docs/devices/2430-100.md

```diff
@@ -17,10 +17,11 @@ description: "Integrate your Gira 2430-100 via Zigbee2mqtt with whatever smart h
 
 ## Notes
 
-None
+How do you pair it?
 
 ## Manual Home Assistant configuration
-Although Home Assistant integration through [MQTT discovery](../integration/home_assistant) is preferred,
+Although Home Assistant integration through [MQTT discovery](../integration/home_assistant) -> 404 
+is preferred,
```

Comment:
How to reproduce it? When I click on it via https://www.zigbee2mqtt.io/devices/GR-ZB01-W.html it works.